### DISCUSSION TITLE
Call FileManager modal

### DISCUSSION THREAD

**Discussion by Crazy-racoon** [luyadev/luya#2146] [Questions & Problems] [answered]
Good afternoon! How can I call the file manager window in the "Custom Angular Controller and Admin View" and call  when choosing a file?

**Comment by nadar** [comment, marked as answer]
You can use all the angularjs directives:

```
<zaa-file-upload model="model" label="someLabel" fieldid="someId"></zaa-file-upload>
```

or only the file modal

```
<storage-file-upload ng-model="model"></storage-file-upload>
```

or only images

```
<storage-image-upload options="options" ng-model="model"></storage-image-upload>
```

sadly, we have no "api" for all the angularjs directives which are available, so you have to take a look at the directives file: https://github.com/luyadev/luya-module-admin/blob/master/src/resources/js/formdirectives.js

Does that answer your question?

**Reply by Crazy-racoon** [reply]
yes, I understood that, but I would like to integrate the file manager into the TinyMCE v5, that is, call the image selection window from TinyMCE, and when choosing a file, so that TinyMCE receives an image

**Reply by nadar** [reply]
And is your TinyMCE implementation angularjs based? Because otherwise this won't work. There is an `storageFileManager` directive https://github.com/luyadev/luya-module-admin/blob/master/src/resources/js/directives.js
